### ISSUE TITLE
styling is affected of components that don't use the marquee when importing 

### ISSUE THREAD

**Issue by silksil** [devnomic/marquee#4]
When I import "import '@devnomic/marquee/dist/index.css'" the styling of components are affected, even those that don't include the Marquee component.

**Comment by laban405** [comment]
Same issue, did you find a solution?

**Comment by Ashish-simpleCoder** [comment]
Yes, because it is importing css file and apply on all elements. We have to use styled-component for this css collison issue to limit the scope of css within that component.

**Comment by injae-kim** [comment]
Fix PR: https://github.com/devnomic/marquee/pull/15
